templates/mobile-notification/order-confirmation.hypr

```diff
@@ -1,2 +1,2 @@
 {% comment %} message length should not exceed 160 characters {% endcomment %}
-{{ labels.mobileOrderConfirmationMessage|safe }}
\ No newline at end of file
+{{ labels.mobileOrderConfirmationMessage|string_format(model.fulfillmentInfo.fulfillmentContact.firstName, model.fulfillmentInfo.fulfillmentContact.lastNameOrSurname, model.siteId)|safe  }}
```

Comment:
According to COM-1937, we need to show the _Retailer Name_, not the shopper's name.